src/content/general/organizations/index.md

```diff
@@ -0,0 +1,145 @@
+---
+linkTitle: Organizations
+title: Organizations
+description: Explaining the organization concept in the Giant Swarm Management API, how to use organizations currently, and our future plans to make use of them.
+last_review_date: 2021-05-07
+weight: 40
+menu:
+  main:
+    parent: general
+user_questions:
+  - What is an organization?
+  - Do I have to use organizations?
+  - What's the benefit of using organizations?
+owner:
+  - https://github.com/orgs/giantswarm/teams/biscuit
+---
+
+# Organizations
+
+<div class="well disclaimer">
+
+<i class="fa fa-warning"></i> This article covers organizations as defined in the [Management API]({{< relref "/ui-api/management-api/_index.md" >}}). These are replacing the organizations as used with the [REST API]({{< relref "/ui-api/rest-api/index.md">}}). Please make sure you read the details about this transition carefully, especially the [roadmap](#roadmap) section, to understand the ramifications for you and your end users.
+
+<!-- TODO: link article about changes and migration -->
+
+</div>
+
+## Introduction
+
+Organizations are a way to manage resources like clusters and apps in a way that different entities are isolated from each other. You can use organizations to separate projects, business units, teams etc.
+
+Clusters are _owned_ by organizations. So in order to be able to manage a cluster (on the Management API level, not on the individual cluster level), access to the organization owning the cluster is required.
```

Comment:
tbh, there's no concept of ownership in a K8s API. It was a fact in our REST API. I don't think Orgs in the K8s API and with that in the MAPI sense should be defined as a means of ownership, cause that gives them a meaning that they cannot really have. E.g. we should be able to move a cluster between orgs (a feature that has been requested a few times in the past) in the MAPI with CAPI.

Anyway, I would suggest explaining it rather along the lines of "a way to help users organize their resources and access control to those resources, which is easier to use than bare RBAC". We can explain that it keeps some of the familiarities of the former org concept but opens it up to be much more flexible and accommodating to more use cases.